functions/transferChatStart.ts

```diff
@@ -136,13 +142,18 @@ export const handler: ServerlessFunctionSignature = TokenValidator(
           resolve(error403("Error: can't transfer to an offline counselor"));
           return;
         }
-      }
 
-      // retrieve attributes of the original task
-      const originalTask = await client.taskrouter
-        .workspaces(context.TWILIO_WORKSPACE_SID)
-        .tasks(taskSid)
-        .fetch();
+        const workerChannel = await client.taskrouter
```

Comment:
I wonder if we should make this call and the call on Line 136 in parallel to save time.  But that can be a future optimization.

Reply:
You are right, I'll change that before merging!